learn/get-started/write-a-graphql-api-with-ballerina.md

```diff
@@ -338,15 +375,37 @@ service /covid19 on new graphql:Listener(9000) {
 }
 ```
 
-Now, you can run this service to serve a GraphQL API to our data source. To run, execute the following command:
+## Run the service
+
+ Execute the command below to run this service to serve a GraphQL API to the data source.
 
 ```shell
 bal run
 ```
 
-> **Note:** The console should have warning logs related to the isolatedness of resources. It is a built-in service concurrency safety feature of Ballerina.
+You view the output below.
+
+```bash
+Compiling source
+	praneesha/graphql:0.1.0
+WARNING [main.bal:(25:5,25:5)] concurrent calls will not be made to this method since the service and the method are not 'isolated'
```

Comment:
Can we improve the code not to have this warning? @shafreenAnfar , @ThisaruGuruge

Reply:
We can, but to do this, we have to add the `isolated` keyword. We decided not to add this since it might be confusing for new users. @shafreenAnfar WDYT?

Reply:
Yeah, adding the word `isolated` is problematic. But we might be able to do it without it, any thoughts? @MaryamZi